cocos/renderer/CCTrianglesCommand.h

```diff
@@ -68,7 +68,7 @@ class CC_DLL TrianglesCommand : public RenderCommand
      @param mv ModelView matrix for the command.
      @param flags to indicate that the command is using 3D rendering or not.
      */
-    void init(float globalOrder, GLuint textureID, GLProgramState* glProgramState, BlendFunc blendType, const Triangles& triangles,const Mat4& mv, uint32_t flags);
+    void init(float globalOrder, GLuint textureID, GLProgramState* glProgramState, BlendFunc blendType, const Triangles& triangles,const Mat4& mv, uint32_t flags, GLuint textureID1 = 0);
```

Comment:
`textureID1 ` here is used for ect1 alpha texture. But the name seems strange. Can it be used for other purpose?